README.md

```diff
@@ -3,88 +3,108 @@
 ![Test](https://github.com/codebuddies/django-concept/workflows/Test/badge.svg)
 [![codecov](https://codecov.io/gh/codebuddies/backend/branch/master/graph/badge.svg)](https://codecov.io/gh/codebuddies/backend)
 
+**Note:** This project is currently _in development_
 
+## Features
 
+- **Auto-reload** - modify the application code in your editor of choice. As you save changes, the application should reload automatically. There should be no need to restart containers to see code changes.
+<!-- TODO -->
 
-Background: https://github.com/codebuddies/codebuddies/issues/1136
+## Reference
 
-The API spec all the proof-of-concepts: https://app.swaggerhub.com/apis-docs/billglover/CodeBuddies/0.0.1
+- [Background]
+- [The API spec all the proof-of-concept]
+- [Crowdsourced brainstorm of problems we want to solve]
+- Will interact as the API supporting [https://github.com/codebuddies/frontend]
 
-Crowdsourced brainstorm of problems we want to solve: https://pad.riseup.net/p/BecKdThFsevRmmG_tqFa-keep
+## Getting started
 
-## Setup
-
-Although it is possible to run this locally, we recommend you run CodeBuddies locally using Docker. We assume you have Docker installed, but if not head on over to the Docker [Getting Started](https://www.docker.com/products/docker-desktop) guide and install Docker for your operating system.
+We recommend you run CodeBuddies locally using Docker. We assume you have Docker installed, but if not head on over to the Docker [Getting Started] guide and install Docker for your operating system.
```

Comment:
The URL "Getting Started" is linking to is missing here as well.

Reply:
The link has been moved to the bottom https://github.com/codebuddies/backend/compare/feature/readme-update#diff-04c6e90faac2675aa89e2176d2eec7d8R233

I've used a reference style link format for readability. See `[link text itself]: http://www.reddit.com` here https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Reply:
Ohh, TIL! Thanks, didn't know it was possible to do this in markdown before, for some reason 😅